content/applications/marketing/marketing_automation.rst

```diff
@@ -1,121 +1,191 @@
+based on predefined filters, triggers and durations of activities.
 
-In the :guilabel:`Target` field, use the drop-down menu to choose which model the target audience
-filters should be based on (e.g. :guilabel:`Contact`, :guilabel:`Lead/Opportunity`,
-:guilabel:`Sales Order`, etc.).
+A new campaign can be created from scratch or from a :ref:`template
+<marketing-automation/campaign-templates>`. To create a campaign, navigate to the
+:menuselection:`Marketing Automation` application from the main Odoo dashboard to open the
+:guilabel:`Campaigns` dashboard. From here, click the :guilabel:`New` button to reveal a new
+campaign form.
 
-Select :guilabel:`Search More...` from the drop-down menu to reveal a :guilabel:`Search: Target`
-pop-up window containing all of the available targeting options.
+.. _marketing-automation/campaign-templates:
 
-Once a :guilabel:`Target` is selected, there's a :guilabel:`Unicity based on` field. This field is
-used to avoid duplicates based on the model chosen in the :guilabel:`Target` field.
+Campaign templates
+------------------
 
-.. example::
-   If :guilabel:`Customers` is chosen as the :guilabel:`Target`, select :guilabel:`Email` in the
-   :guilabel:`Unicity based on` field so Odoo only processes one record for each customer email
-   address.
+Odoo provides six campaign templates to help users get started. The campaign template cards only
+display while there are no existing campaigns in the database. Once a campaign has been created, the
+template cards on the *Campaigns* dashboard are replaced with a Kanban view of the existing
+campaigns.
 
-Select :guilabel:`Search More...` from the :guilabel:`Unicity based on` drop-down menu to reveal all
-available options in a pop-up window.
+To get started with a template, navigate to the :menuselection:`Marketing Automation` application
+from the main Odoo dashboard to open the :guilabel:`Campaigns` dashboard, which displays six
+campaign template cards:
 
-Last on the campaign form is the :guilabel:`Filter` field. This is where more specific targeting
-options can be layered into the campaign to further narrow the number and type of recipients that
-receive the marketing automation material.
+- | :guilabel:`Tag Hot Contacts`
+  | :guilabel:`Send a welcome email to contacts and tag them if they click it.`
+- | :guilabel:`Welcome Flow`
+  | :guilabel:`Send a welcome email to new subscribers, remove the address that bounced.`
+- | :guilabel:`Double Opt-in`
+  | :guilabel:`Send an email to new recipients to confirm their consent.`
+- | :guilabel:`Commercial prospection`
+  | :guilabel:`Send a free catalog and follow-up according to reactions.`
+- | :guilabel:`Schedule Calls`
+  | :guilabel:`If a lead is created for existing contact, schedule a call with their salesperson.`
+- | :guilabel:`Prioritize Hot leads`
+  | :guilabel:`Send an email to new leads and assign them a high priority if they open it.`
```

Comment:
purely a suggestion here, but it might be nice to add a screenshot here, so viewers know what these template cards look like. But, again, not a requirement ... totally up to you! 👍